### PR TITLE
fix: TT-261 handle response Not found 

### PR DESCRIPTION
**Pull request by kevinjlope**
handle response Not found, and added unit testing of method add_complementary_info

### CODE REVIEW

File: tests/time_tracker_api/time_entries/time_entries_repository_test.py [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+        == expected_projects.__dict__['customer_id']
+    )
+    assert (
+        expected_time_entry.__dict__['customer_name']
+        == expected_projects.__dict__['customer_name']
+    )
+    assert (
+        expected_time_entry.__dict__['activity_name']
+        == expected_activities.__dict__['name']
+    )
```
Comment (jcalarcon98) [inline tests/time_tracker_api/time_entries/time_entries_repository_test.py:89]: I think that you are not making the test corresponding to function `add_add_complementary_info`, actually, you are only making the call to the functions that are inside.

**Review by jcalarcon98 — COMMENTED** [review]
Good Job Kevin, you can make those small changes and the approval can be made.

**Review by jcalarcon98 — APPROVED** [review]
Good Job!

File: tests/time_tracker_api/time_entries/time_entries_repository_test.py [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+        'id': 'project_id1',
+        'name': 'project_name',
+        'description': 'do some testing',
+        'project_type_id': "id2",
+        'tenant_id': 'tenantid1',
+    }
+    return project_data
+
+
+def create_activity_data():
```
[the remark this answers is not in the document]
Comment (kevinjlope) [reply tests/time_tracker_api/time_entries/time_entries_repository_test.py]: done.

File: tests/time_tracker_api/time_entries/time_entries_repository_test.py [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+    )
+    activities_db_get_all_mock = mocker.patch.object(
+        ActivityCosmosDBDao, 'get_all'
+    )
+    users_mock = mocker.patch.object(AzureConnection, 'users')
+
+    expected_project = ProjectCosmosDBModel(create_project_data())
+    expected_activity = ActivityCosmosDBModel(create_activity_data())
+    expected_time_entry_in = TimeEntryCosmosDBModel(create_time_entry_data())
+    expected_users = AzureUser('email1', [], 'id', 'name', ['admin'])
```
[the remark this answers is not in the document]
Comment (kevinjlope) [reply tests/time_tracker_api/time_entries/time_entries_repository_test.py]: done.

**Review by scastillo-jp — APPROVED** [review]

**Pull request merged**